CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ internal API changes are not present.
 v0.35.0-rc.1 (2023-07-17)
 -------------------------
 
+v0.35.0-rc.1 (2023-07-17)
```

Comment:
I think this header might be duplicated?

Reply:
Thank you